lib/plugins/track.js

```diff
@@ -50,6 +50,7 @@ function configureValues (track) {
       return getMeta('description')
     },
     td_url: function () {
+      if (!document.location || !document.location.href) return ''
```

Comment:
Nit: maybe worth adding a `{}` to match the style everywhere else (or make it a ternary)